docs/source/types.rst

```diff
@@ -16,8 +16,72 @@ Available content types in a Plone site can be listed and queried by accessing t
    :language: http
 
 
+The API consumer can create, read, update, and delete a content-types schema.
+
+======= =============================================== =====================================================================
+Verb    URL                                             Action
+======= =============================================== =====================================================================
+POST    ``/@types/Document``                            Add field/fieldset to content type schema
```

Comment:
Maybe we should parameterize "Document" here to make it clear this works for all content types. The other option would be to mention that this example is for the Document type and that this works similarly for other types.